datacapturing/src/cyface/java/de/cyface/datacapturing/CyfaceDataCapturingService.java

```diff
@@ -154,6 +155,25 @@ public void startWifiSurveyor() throws SetupException {
         }
     }
 
+    /**
```

Comment:
that's the same code as MovebisDataCapturingService.deregisterJwtToken executes, which was what I wanted to test in our app to reproduce and fix bugs. That's why I added the logout functionality to our app, which calls this method here.